components/admin_console/system_users/list/system_users_list.jsx

```diff
@@ -16,10 +14,7 @@ import ResetPasswordModal from 'components/admin_console/reset_password_modal';
 import SearchableUserList from 'components/searchable_user_list/searchable_user_list.jsx';
 import UserListRowWithError from 'components/user_list_row_with_error.jsx';
 
-import SystemUsersDropdown from './system_users_dropdown.jsx';
-
-const dispatch = store.dispatch;
-const getState = store.getState;
+import SystemUsersDropdown from '../system_users_dropdown.jsx';
 
 export default class SystemUsersList extends React.Component {
```

Comment:
Please add component test for `SystemUsersList`.

Reply:
Done!